cypress/e2e/settings/data-import/check-orders-record-type-on-create-edit-field-mapping-profile.cy.js

```diff
@@ -0,0 +1,148 @@
+import { Permissions } from '../../../support/dictionary';
+import {
+  SettingsDataImport,
+  FieldMappingProfiles,
+} from '../../../support/fragments/settings/dataImport';
+import { SETTINGS_TABS } from '../../../support/fragments/settings/dataImport/settingsDataImport';
+import SettingsMenu from '../../../support/fragments/settingsMenu';
+import Users from '../../../support/fragments/users/users';
+import getRandomPostfix from '../../../support/utils/stringTools';
+
+describe('Data Import', () => {
+  const testData = {
+    profileName: `autotest_mapping_profile_name_${getRandomPostfix()}`,
+    existingRecordTypes: [
+      'Select FOLIO record type (disabled)',
+      'Instance',
+      'Holdings',
+      'Item',
+      'Order',
+      'Invoice',
+      'MARC Bibliographic',
+      'MARC Holdings (disabled)',
+      'MARC Authority',
+    ],
+    user: {},
+  };
+
+  before('Create test data', () => {
+    cy.createTempUser([Permissions.settingsDataImportEnabled.gui]).then((userProperties) => {
+      testData.user = userProperties;
+    });
+  });
+
+  beforeEach('Login', () => {
+    cy.login(testData.user.username, testData.user.password, {
+      path: SettingsMenu.dataImportSettingsPath,
+      waiter: SettingsDataImport.waitLoading,
+    });
+  });
+
+  after('Delete test data', () => {
+    cy.getAdminToken().then(() => {
+      Users.deleteViaApi(testData.user.userId);
+    });
+  });
+
+  describe('Settings', () => {
+    it(
+      'C365130 Verify Orders record type on creating new field mapping profile (folijet) (TaaS)',
+      { tags: ['extendedPath', 'folijet'] },
+      () => {
+        // Go to Settings application-> Data import-> Field mapping profiles
+        SettingsDataImport.selectSettingsTab(SETTINGS_TABS.FIELD_MAPPING_PROFILE);
+
+        // Click Actions button, Select New field mapping profile
+        const FieldMappingProfileEditForm =
+          FieldMappingProfiles.clickCreateNewFieldMappingProfile();
+
+        [
+          { incomingRecordType: 'MARC Bibliographic', existingRecordType: 'Order' },
+          { incomingRecordType: 'MARC Authority', existingRecordType: 'Order' },
+          { incomingRecordType: 'EDIFACT invoice', existingRecordType: 'Order' },
+        ].forEach(({ incomingRecordType, existingRecordType }) => {
+          // In "Incoming record type" dropdown select option
+          // In "Folio record type" dropdown list select option
+          FieldMappingProfileEditForm.fillMappingProfileFields({
+            name: testData.profileName,
+            incomingRecordType,
+            existingRecordType,
+          });
+
+          // Check the contents of the "FOLIO record type" dropdown list
+          FieldMappingProfileEditForm.getDropdownOptionsList({ label: 'FOLIO record type' }).then(
+            (optionsList) => {
+              cy.expect(optionsList).to.eql(testData.existingRecordTypes);
```

Comment:
can we create a new fragment method for this part ?

Reply:
Done